core/controllers/question_editor.py

```diff
@@ -115,22 +115,31 @@ class QuestionSkillLinkHandler(base.BaseHandler):
     """A handler for linking and unlinking questions to or from a skill."""
 
     @acl_decorators.can_manage_question_skill_status
-    def put(self, question_id, skill_id):
+    def put(self, question_id, skill_ids):
         """Updates the difficulty of the question with respect to the given
```

Comment:
Docstring is wrong (should be plural?) Ditto below.

Reply:
Done